config.go

```diff
@@ -62,6 +62,7 @@ func (c Config) Validate() error {
 		return errors.New("Must provide a non-empty StateEncryptionKey")
 	}
 
+	c.RDSConfig.FillDefaults()
```

Comment:
Hi Joshua, thanks for updating PR. The new `FillDefaults` method won't work here either as we are still passing a value here. We can change it to
```func (c *Config) Validate() error {```
and it should work.